Assets/AudioManager.cs

```diff
@@ -0,0 +1,70 @@
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager Instance { get; private set; }
+
+    [SerializeField] private BgmManger bgmManager;
+    [SerializeField] private SEManager seManager;
+
+    void Awake()
+    {
+        if (Instance == null)
+        {
+            Instance = this;
+            DontDestroyOnLoad(gameObject);
+        }
+        else
+        {
+            Destroy(gameObject);
+        }
+    }
+
+    void Start()
+    {
+        if (bgmManager == null)
+            bgmManager = FindObjectOfType<BgmManger>();
+        
+        if (seManager == null)
+            seManager = FindObjectOfType<SEManager>();
+    }
+
+    public void UpdateAllVolumeSettings()
+    {
+        if (bgmManager != null)
+            bgmManager.UpdateVolumeSettings();
+        
+        if (seManager != null)
+            seManager.UpdateVolumeSettings();
```

Comment:
```suggestion
        if (bgmManager)
            bgmManager.UpdateVolumeSettings();
        
        if (seManager)
            seManager.UpdateVolumeSettings();
```
`==null`や`!= null`は不要で、これで大丈夫です！